### PR TITLE
Fix security alerts

### PR DESCRIPTION
**Pull request by kse3hi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,21 +19,22 @@
...
     "nyc": "^15.1.0",
     "prettier": "^2.7.1",
     "ts-node": "^10.9.1",
-    "typescript": "^4.8.3"
+    "typescript": "^4.8.3",
+    "@babel/traverse": "^7.23.2"
   },
   "dependencies": {
-    "axios": "^0.27.2",
+    "axios": "^1.6.7",
```
Comment (dennismeister93) [inline package.json]: thats not working because there are multiple major changes inside, please keep at 0.27

**Review by dennismeister93 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (kse3hi) [reply package.json]: As for security alert, https://github.com/eclipse-velocitas/velocitas-project-generator-npm/security/dependabot/3 we need to calculate effort to bulk version or calculate and mitigate related risks

Comment (dennismeister93) [reply package.json]: how does this issue affect us particulary in which use case?

File: package-lock.json [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+  "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "@eclipse-velocitas/velocitas-project-generator",
       "version": "3.0.1",
       "license": "Apache-2.0",
       "dependencies": {
-        "axios": "^0.27.2",
+        "axios": "^1.6.7",
```
Comment (doosuu) [inline package-lock.json]: Axios was deliberately kept on this low version. @dennismeister93  would need to perform additional testing before pulling up the version to this new one.

File: package.json [hunk 3]
```diff
@@ -19,21 +19,22 @@
...
+    "chai": "^4.3.10",
     "chai-as-promised": "^7.1.1",
     "mocha": "^10.0.0",
     "nock": "^13.2.9",
     "nyc": "^15.1.0",
     "prettier": "^2.7.1",
     "ts-node": "^10.9.1",
-    "typescript": "^4.8.3"
+    "typescript": "^4.8.3",
+    "@babel/traverse": "^7.23.2"
```
Comment (doosuu) [inline package.json]: A whole new dependency? Where does it come from?

File: package.json (hunk 1 above)
Comment (doosuu) [reply package.json]: We had troubles with this new major axios version in the past.

File: package.json (hunk 3 above)
Comment (kse3hi) [reply package.json]: It was a sub-sub-sub dependency that was vulnerable. 
But with a re-generated version of the `package-lock.json` issue was solved.

File: package-lock.json (hunk 2 above)
Comment (kse3hi) [reply package-lock.json]: Without bulking a version, we will keep 2 moderate vulnerability.
I will create follow up story and will link those 2 risks as dependency.

File: package.json (hunk 1 above)
Comment (kse3hi) [reply package.json]: Will be tracked in https://github.com/eclipse-velocitas/velocitas-project-generator-npm/issues/34

**Comment by kse3hi** [comment]
Self finding: NOTICE_3rd_PARTY.... is totally misaligned with real dependencies - follow up https://github.com/eclipse-velocitas/velocitas-project-generator-npm/issues/33

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**